.template/variants/web/package.json.rb

```diff
@@ -37,9 +37,16 @@
   '--no-source-map',
   '--load-path=node_modules'
 ]
+production_bundled_stylesheet_options = bundled_stylesheet_options + [
```

Comment:
We might need to rename the other variable:

```ruby
bundled_stylesheet_options = [
  '--no-source-map',
  '--load-path=node_modules'
]
```

To:

```ruby
bundled_stylesheet_base_options = [
  '--no-source-map',
  '--load-path=node_modules'
]
```

It looks more descriptive:

```suggestion
production_bundled_stylesheet_options = bundled_stylesheet_base_options + [
```